app/frontend/src/features/messages/constants.ts

```diff
@@ -24,12 +22,15 @@ export const NO_GROUP_CHAT = "No group chats yet";
 export const NO_MESSAGES = "No messages";
 export const TITLE = "Title";
 
-export interface HostRequestListItemProps {
-  hostRequest: HostRequest.AsObject;
-}
-
-export const nightsRequested = ({
-  toDate,
-  fromDate,
-}: HostRequest.AsObject) => `Requesting to stay for
-${numNights(toDate, fromDate)}`;
+export const hostingStatusText = (isHost: boolean, status: number) => {
+  if (status === 0) {
+    return `This request is still pending`;
+  } else
+    return isHost
+      ? `You have ${Object.values(hostRequestStatusLabels)[
+          status
+        ].toLowerCase()} this request`
+      : `Your request was ${Object.values(hostRequestStatusLabels)[
+          status
+        ].toLowerCase()}`;
+};
```

Comment:
```suggestion
export const hostingStatusText = (
  isHost: boolean,
  status: HostRequestStatus
) => {
  if (status === HostRequestStatus.HOST_REQUEST_STATUS_PENDING) {
    return `This request is still pending`;
  } else
    return isHost
      ? `You have ${hostRequestStatusLabels[status].toLowerCase()} this request`
      : `Your request was ${hostRequestStatusLabels[status].toLowerCase()}`;
};
```

Reply:
Yep, definitely better, I was trying to do somethink like that but couldn't figure it out without TS yelling at me 😂